pkg/actions/cluster/delete.go

```diff
@@ -29,6 +28,11 @@ import (
 	"github.com/kris-nova/logger"
 )
 
+type NodeGroupDrainer interface {
+	Drain(nodeGroups []eks.KubeNodeGroup, plan bool, maxGracePeriod, nodeDrainWaitPeriod time.Duration, undo, disableEviction bool) error
+}
+type VpcCniDeleter func(clusterName string, ctl *eks.ClusterProvider, clientSet kubernetes.Interface)
```

Comment:
We don't need to export this.
```suggestion
type vpcCNIDeleter func(clusterName string, ctl *eks.ClusterProvider, clientSet kubernetes.Interface)
```

Reply:
Changed